adapters/celestia/Cargo.toml

```diff
@@ -13,7 +13,7 @@ tendermint = "0.32"
 tendermint-proto = "0.32"
 
 # Convenience
-anyhow = "1.0.62"
+anyhow = "1.0.68"
```

Comment:
Is there a `workspace` version of anyhow we can use instead?

Reply:
Good point!